web/ui/src/Health/healthIconDirective.js

```diff
@@ -6,8 +6,10 @@
     'use strict';
 
     angular.module('healthIcon', [])
-    .directive("healthIcon", [
-    function(){
+    .directive("healthIcon", ["hcStatus",
+    function(hcStatus){
+        let {OK, FAILED, NOT_RUNNING, UNKNOWN} = hcStatus;
```

Comment:
I did not implement it this time around because it was not present before, and I wanted to minimize the changes. it should definitely be implemented in the future.